Pass `counterparty_node_id` to `fail_holding_cell_htlcs`

ViktorTigerstrom opened the pull request:
Both #1507 and #1403 needs the `counterparty_node_id` to be passed to `ChannelManager::fail_holding_cell_htlcs`. This PR simply passes the parameter to the function, so that both PRs can be based on this.

tnull commented on lightning/src/ln/channelmanager.rs:
```diff
@@ -5066,7 +5066,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					Ok((commitment_opt, holding_cell_failed_htlcs)) => {
 						if !holding_cell_failed_htlcs.is_empty() {
-							failed_htlcs.push((holding_cell_failed_htlcs, *channel_id));
+							failed_htlcs.push((holding_cell_failed_htlcs, *channel_id, chan.get_counterparty_node_id()));
```
nit: same here: wrap at 100 chars.

tnull reviewed (dismissed):
LGTM, only two tiny formatting nits.

ViktorTigerstrom commented:
Thanks for the review @tnull!
Addressed the feedback with cd59bc8! Not completely sure if it really made things better though as the code now became a bit less consistent with how we wrap the code in rest of the `ChannelManager` codebase :).

tnull commented:
> Addressed the feedback with [cd59bc8](https://github.com/lightningdevkit/rust-lightning/commit/cd59bc83dee4ee6e76d83478212d07488b791089)! Not completely sure if it really made things better though as the code now became a bit less consistent with how we wrap the code in rest of the `ChannelManager` codebase :).

Yeah, I think we currently follow the policy to slowly improve the quality and formatting in parts we touch. Hopefully, though, we may be able to adopt `rustfmt` soon, which then hopefully would make formatting nits go away..

ViktorTigerstrom commented:
> Yeah, I think we currently follow the policy to slowly improve the quality and formatting in parts we touch. Hopefully, though, we may be able to adopt `rustfmt` soon, which then hopefully would make formatting nits go away..

Ok great thanks :)! Squashed the line wrapping fixup without changes.
